docs/examples/util/run-all-complete-examples.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
Agreed. 

1. I'll add the test you are talking about as part of: https://jira.numenta.com/browse/NUP-2401  (Fix results discrepancy between OPF, network API and algorithm API)
2. I'll take out the scripts under `docs/util`.